charts/appflowy/Chart.yaml

```diff
@@ -31,3 +31,6 @@ dependencies:
   - name: appflowy-web
     condition: appflowy-web.enabled
     version: 0.1.0
+  - name: appflowy-ai
+    condition: global.ai.enabled
```

Comment:
I am thinking whether we should use ai.enabled instead of global.ai.enabled for consistency, but for the time being, we can keep it this way.